src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -232,6 +238,12 @@ export class OperationDetails {
         if (operation) {
             await this.loadDefinitions(operation);
             if (this.showExamples) this.parseExamples(operation);
+
+
+            if(!operation) return null;
```

Comment:
If operation is not changed in the functions then it will always not be falsy. See line 238.

Reply:
true, thanks. left things after testing :)